modules/olmv1-version-range-comparisons.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-version-range-comparisons_{context}"]
+= Version comparison strings
+
+You define a version range by adding a comparison string to the `spec.version` field in an Operator or extension's custom resource (CR). A comparison string is a list of space- or comma-separated values and one or more comparison operators. You can add an another comparison string by including an `OR` (`||`) comparison operator between the strings.
+
+.Basic comparisons
+[options="header"]
+|===
+
+|Comparison operator |Definition
+
+|`=`
+|Equal to
+
+|`!=`
+|Not equal to
+
+|`>`
+|Greater than
+
+| `<`
```

Comment:
No slash required here?

Reply:
The slash is only required when I use `<=`. By default, the asciidoctor markup language turns `<=` into an arrow.